src/components/datagrid/body/header/column_actions.test.tsx

```diff
@@ -0,0 +1,496 @@
+            "className": "",
+            "color": "text",
+            "iconType": "sortDown",
+            "isDisabled": false,
+            "label": <EuiI18n
+              default="Sort {schemaLabel}"
+              token="euiColumnActions.sort"
+              values={
+                Object {
+                  "schemaLabel": <EuiI18n
+                    default="Z-A"
+                    token="euiColumnSortingDraggable.defaultSortDesc"
+                  />,
+                }
+              }
+            />,
+            "onClick": [Function],
+            "size": "xs",
+          }
+        `);
+      });
+
+      it('calls onSort on click', () => {
+        callActionOnClick(sortAsc);
+        expect(onSort).toHaveBeenCalledWith([{ direction: 'asc', id: 'B' }]);
+
+        callActionOnClick(sortDesc);
+        expect(onSort).toHaveBeenCalledWith([{ direction: 'desc', id: 'B' }]);
+      });
+    });
+
+    describe('when active sorts are present', () => {
+      describe('when current column is sorting by asc', () => {
+        const items = getColumnActions({
+          ...testArgs,
+          sorting: { onSort, columns: [{ id: 'B', direction: 'asc' }] },
+        });
+        const sortAsc = items[1];
+
+        it('renders sortAsc as selected', () => {
+          expect(sortAsc.className).toEqual(
+            'euiDataGridHeader__action--selected'
+          );
+        });
+
+        it('unsets the current sort if sortAsc is clicked again', () => {
+          callActionOnClick(sortAsc);
+          expect(onSort).toHaveBeenCalledWith([]);
+        });
+      });
+
+      describe('when current column is sorting by desc', () => {
+        const items = getColumnActions({
+          ...testArgs,
+          sorting: { onSort, columns: [{ id: 'B', direction: 'desc' }] },
+        });
+        const sortAsc = items[1];
+        const sortDesc = items[2];
+
+        it('renders sortDesc as selected', () => {
+          expect(sortDesc.className).toEqual(
+            'euiDataGridHeader__action--selected'
+          );
+        });
+
+        it('sets a new sort if the direction is changed', () => {
+          callActionOnClick(sortAsc);
+          expect(onSort).toHaveBeenCalledWith([{ id: 'B', direction: 'asc' }]);
+        });
+      });
+    });
+
+    describe('modifies the sort label based on schema', () => {
+      it('renders low-high instead of A-Z for number schema', () => {
+        const items = getColumnActions({
+          ...testArgs,
+          column: { id: 'A' },
+          schema: { A: { columnType: 'numeric' } },
+          sorting: { onSort, columns: [{ id: 'A', direction: 'desc' }] },
+        });
+        const sortAscLabel = items[1].label as ReactElement;
+        const sortDescLabel = items[2].label as ReactElement;
+
+        expect(sortAscLabel.props.values.schemaLabel.props.default).toEqual(
+          'Low-High'
+        );
+        expect(sortDescLabel.props.values.schemaLabel.props.default).toContain(
```

Comment:
Is there extra content here that disallows using `toEqual`?

Reply:
Whoops, not at all, I think I just brainfarted on this (I was originally trying to assert on the text content but it wasn't sending back what I wanted, so I changed it to dive into props and forgot to change back to `toEqual`. Thanks Greg for the awesome catch!

```suggestion
        expect(sortDescLabel.props.values.schemaLabel.props.default).toEqual(
```